test/test_samples.py

```diff
@@ -0,0 +1,48 @@
+# pylint: disable=import-outside-toplevel
+import os
+
+
+class Cd:
+    def __init__(self, new_path):
+        self.new_path = new_path
+        self.saved_path = None
+
+    def __enter__(self):
+        self.saved_path = os.getcwd()
+        os.chdir(str(self.new_path))
+
+    def __exit__(self, etype, value, traceback):
+        os.chdir(str(self.saved_path))
+
+
+def test_capture_from_file(sample_data_file):
+    import subprocess
+    from pathlib import Path
+
+    current_working_directory = Path(os.getcwd()).resolve()
+    sample = (
+        current_working_directory
+        / ".."
+        / ".."
+        / "samples"
+        / "sample_capture_from_file.py"
+    )
+
+    with Cd(sample_data_file.parent):
+        subprocess.check_output(args=("python", str(sample),))
+
+
+def test_print_version_info():
+    import subprocess
+    from pathlib import Path
+
+    current_working_directory = Path(os.getcwd()).resolve()
+    sample = (
+        current_working_directory
+        / ".."
+        / ".."
+        / "samples"
+        / "sample_print_version_info.py"
+    ).resolve()
+
+    subprocess.check_output(args=("python", str(sample),))
```

Comment:
The big block in the middle looks like it is duplicated from above except for a minor difference that is probably a copy-paste error. Can this be made into a reusable function?

```suggestion
def test_capture_from_file(sample_data_file):
    run_sample(
        name="capture_from_file",
        working_directory=sample_data_file.parent
    )

def test_print_version_info():
    run_sample(
        name="print_version_info"
    )
```